plugins/apache_http_logs.yaml

```diff
@@ -0,0 +1,182 @@
+version: 0.0.1
+title: Apache HTTP Server
+description: Log parser for Apache HTTP Server
+parameters:
+  - name: log_format
+    description: When choosing the 'default' option, the agent will expect and parse logs in a format that matches the default logging configuration. When choosing the 'observIQ' option, the agent will expect and parse logs in an optimized JSON format that adheres to the observIQ specification, requiring an update to the apache2.conf file. See the Apache HTTP Server source page for more information.
+    type: string
+    supported:
+      - default
+      - observiq
+    default: default
+  - name: enable_access_log
+    description: Enable to collect Apache HTTP Server access logs
+    type: bool
+    default: true
+  - name: access_log_path
+    description: Path to access log file
+    type: "[]string"
+    default: 
+      - "/var/log/apache2/access.log"
+  - name: enable_error_log
+    description: Enable to collect Apache HTTP Server error logs
+    type: bool
+    default: true
+  - name: error_log_path
+    description: Path to error log file
+    type: "[]string"
+    default: 
+      - "/var/log/apache2/error.log"
+  - name: start_at
+    description: Start reading file from 'beginning' or 'end'
+    type: string
+    supported:
+     - beginning
+     - end
+    default: end
+
+template: |
+  receivers:
+  {{ if .enable_access_log }}
+    filelog/access_log:
+      include:
+        # {{ range $fp := .access_log_path }}
+        - '{{ $fp }}'
+        # {{end}}
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'apache_http.access'
+      operators:
+        {{ if eq .log_format "default" }}
+        - id: access_regex_parser
+          type: regex_parser
+          regex: '^(?P<remote_addr>[^ ]*) (?P<remote_host>[^ ]*) (?P<remote_user>[^ ]*) \[(?P<timestamp>[^\]]*)\] "(?P<method>\S+) +(?P<path>[^ ]*)( (?P<protocol>[^/]*)/(?P<protocol_version>[^\"]*)|[^\"]*)?" (?P<status>[^ ]*) (?P<body_bytes_sent>[^ ]*)(?: "(?P<http_referer>[^\"]*)" "(?P<http_user_agent>[^\"]*)"(?:\s+(?P<http_x_forwarded_for>[^ ]+))?)?'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%d/%b/%Y:%H:%M:%S %z'
+          severity:
+            parse_from: attributes.status
+            preset: none
+            preserve_to: attributes.status
```

Comment:
remove `preserve_to` (there are some other instances of it being used in this plugin), this is default behavior in OTel now